boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -198,13 +195,24 @@ public void processOpts() {
             () -> additionalProperties.put(BUILD_DIST, "dist"));
     }
 
-    private void applyAngularVersion(SemVer angularVersion) {
-        if (!angularVersion.atLeast("10.0.0")) {
-            throw new IllegalArgumentException("Only angular versions >= 10.0.0 are supported.");
-        }
+    private void applyAngularVersion(String versionRange) {
```

Comment:
I think this overload of `applyAngularVersion` is skipping the >= 10 check completely.
I'm not sure which version of the range it should check though? (the smallest, maybe, since it won't actually work with that version)